marketplaceItems/soloPoint/appResources/appData/resources/js/soloPointTablet.js

```diff
@@ -0,0 +1,130 @@
+/*
+    Solo Point
+    Created by Milad Nazeri on 2019-01-07
+    Copyright 2016 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+    Point to solo someone to hear them better in a crowd!
+    TABLET JS
+    
+*/
+
+
+// *************************************
+// START UTILITY FUNCTIONS
+// *************************************
+// #region Utilty
+
+var debug = true;
+var FALSE = "false";
+
+// Easy log function checking for message, an object to stringify, and whether it should be enabled or not
+function log(message, object, enabled){
```

Comment:
is this function (and all of its callers) necessary to keep in production code?

Reply:
The only problem I see with that is if I need it for logging and having to bring the code back.  Let me know if you have some thoughts on that.  Could put it in a module instead?

Reply:
A module might be OK. Logging code is usually pretty specific. It's probably best to save this code snippet for yourself and pull it out when necessary.